source/fe/fe_simplex_p.cc

```diff
@@ -561,9 +562,15 @@ FE_SimplexPoly<dim, spacedim>::face_to_cell_index(
       const unsigned int index =
         face_dof_index - this->get_first_face_quad_index(face);
 
-      // the same is true here as above for the 3d case -- someone will
-      // just have to draw a bunch of pictures. in the meantime,
-      // we can implement the degree <= 3 case in which it is simple
+      // Since polynomials on simplices are typically defined in barycentric
+      // coordinates, handling higher degree elements is not that hard: if the
+      // DoFs come in triples then they simply need to be rotated in the correct
+      // way (as though they were the vertices). However, we don't implement
+      // higher than cubic elements, so that is not yet done here. For example,
+      // for a P4 element a quad will have 3 DoFs, and each of those DoFs will
```

Comment:
Do you mean to say "face" here instead of "quad"? I know that we have functions that say "quad" for historical reasons, but it's certainly confusing here.

Reply:
We really need better names for these, but in this context I prefer 'quad' since in some contexts "dofs per face" can include lower-dimensional objects whereas "dofs per quad" never does.